memo-app/src/styles.css

```
@@ -35,3 +35,8 @@ button {
 button.delete-button {
   margin-left: 10px;
 }
+
+.login-button {
+  text-align: right;
```

center で良さそう？と思いましたが理由があったら教えてください！

```suggestion
  text-align: center;
```

---

`right`にしたほうがボタンの位置は適切かと考えていましたが、画面幅を変えた時に不自然ですね...
ご指摘いただいたとおり`center`に修正しました。